### DISCUSSION TITLE
Limited recipes to unlimited BOP

### DISCUSSION THREAD

**Discussion by Sherincal94** [chromiecraft/chromiecraft#4647] [Suggestions]
I suggest to transform limited supply recipes from limited supply BOE to unlimited BOP to prevent vendor camping and exploiting.

The situation now is that several vendors have critical item recipes in limited supply with very long respawn timer (12h) which is exploited by vendor camping the recipe by certain players and then posting the recipe on AH for high amounts.

Campers can in this way prevent the recipe from being bought by an average player for weeks since they cant compete with the camper nor use any script to quickly buy the recipe.

The change would ensure smoother experience and reduce vendor camping.

**Comment by daemonhart** [comment]
Why bother making them BoP at that point? I feel it either ought to remain as originally done (as-is) or simply fully unlimited, with a strong preference for the former.

**Comment by Peppersprayz** [comment]
With this, aldor tailors will be f*cked because you need to buy a recip from scryers to be able to level the prof, and the only way to get it is buying it with alt.
It would be way easier to just make limited sells unlimited or faster the respawn

**Comment by Frodhi** [comment]
This needs fixing.
It promotes scalper mentality and it hinders the gameplay of others.

**Comment by trinitarius** [comment]
I wouldn't say they should make them both unlimited and BoP. Either unlimited or BoP but limited, one of those should fix people messing up the economy even more as it already is due to population.

**Comment by cachocarne** [comment]
My Blacksmither is standing in front of Aaron Hollman since days, waiting for the plans for the Adamantite Rod so my Enchanter can continue progressing. I still haven't even seen it, there's people coming all the time and recipe is not there. Unsurprisingly, the 4g recipe is a few times on the AH at 220g sold by the same player, who controls the cd and is farming this and other limited recipes. This is not "business", it's not "fun", it's just douchebag toxic mentality on an otherwise friendly server. Is it necessary to do this, really? I doubt this player even needs the money, but even if it was the case… is it worth to f**k other players and keep them unable to progress?
